src/category_theory/eq_to_hom.lean

```diff
@@ -141,6 +143,43 @@ lemma congr_hom {F G : C ⥤ D} (h : F = G) {X Y} (f : X ⟶ Y) :
   F.map f = eq_to_hom (congr_obj h X) ≫ G.map f ≫ eq_to_hom (congr_obj h Y).symm :=
 by subst h; simp
 
+
+section heq
+
+/- Composition of functors and maps w.r.t. heq -/
+
+variables
+  {E : Type u₁} [category.{v₁} E]
+  {F G : C ⥤ D} {x y z : C} {f : x ⟶ y} {g : y ⟶ z}
```

Comment:
- We generally use capital letters for objects
- We need to be careful here to be as universe-polymorphic as possible so that the results are as general as possible. Hence, we need to introduce new universe variables for the objects and morphisms of `E`.

```suggestion
variables {E : Type u₃} [category.{v₃} E] {F G : C ⥤ D} {X Y Z : C} {f : X ⟶ Y} {g : Y ⟶ Z}
```

For this, you will need to declare the new universe variables at the top of the file:
```lean
-- morphism levels before object levels. See note [category_theory universes].
universes v₁ v₂ v₃ u₁ u₂ u₃
```